src/SonarQube.Client/Services/SonarQubeClient.cs

```diff
@@ -89,68 +107,67 @@ public Task<Result<ServerIssue[]>> GetIssuesAsync(ConnectionRequest connection,
+        public Task<Result<OrganizationResponse[]>> GetOrganizationsAsync(OrganizationRequest request, CancellationToken token)
         {
             const string OrganizationsAPI = "api/organizations/search"; // Since 6.2; internal
 
             var query = AppendQueryString(OrganizationsAPI, "?p={0}&ps={1}", request.Page, request.PageSize); // TODO: should handle optional params
 
-            return InvokeSonarQubeApi(connection, query, token,
+            return InvokeSonarQubeApi(query, token,
                 stringResponse => JObject.Parse(stringResponse)["organizations"].ToObject<OrganizationResponse[]>());
         }
 
-        public Task<Result<PluginResponse[]>> GetPluginsAsync(ConnectionRequest connection, CancellationToken token)
+        public Task<Result<PluginResponse[]>> GetPluginsAsync(CancellationToken token)
         {
             const string ServerPluginsInstalledAPI = "api/updatecenter/installed_plugins"; // Since 2.10; internal
 
-            return InvokeSonarQubeApi(connection, ServerPluginsInstalledAPI, token,
+            return InvokeSonarQubeApi(ServerPluginsInstalledAPI, token,
                 stringResponse => ProcessJsonResponse<PluginResponse[]>(stringResponse, token));
         }
 
-        public Task<Result<ProjectResponse[]>> GetProjectsAsync(ConnectionRequest connection, CancellationToken token)
+        public Task<Result<ProjectResponse[]>> GetProjectsAsync(CancellationToken token)
         {
             const string ProjectsAPI = "api/projects/index"; // Since 2.10
 
-            return InvokeSonarQubeApi(connection, ProjectsAPI, token,
+            return InvokeSonarQubeApi(ProjectsAPI, token,
                 stringResponse => ProcessJsonResponse<ProjectResponse[]>(stringResponse, token));
         }
 
-        public Task<Result<PropertyResponse[]>> GetPropertiesAsync(ConnectionRequest connection, CancellationToken token)
+        public Task<Result<PropertyResponse[]>> GetPropertiesAsync(CancellationToken token)
         {
             const string PropertiesAPI = "api/properties/"; // Since 2.6
 
-            return InvokeSonarQubeApi(connection, PropertiesAPI, token,
+            return InvokeSonarQubeApi(PropertiesAPI, token,
                 stringResponse => ProcessJsonResponse<PropertyResponse[]>(stringResponse, token));
         }
 
-        public Task<Result<QualityProfileChangeLogResponse>> GetQualityProfileChangeLogAsync(ConnectionRequest connection,
+        public Task<Result<QualityProfileChangeLogResponse>> GetQualityProfileChangeLogAsync(
             QualityProfileChangeLogRequest request, CancellationToken token)
         {
             const string QualityProfileChangeLogAPI = "api/qualityprofiles/changelog"; // Since 5.2
 
             var query = AppendQueryString(QualityProfileChangeLogAPI, "?profileKey={0}&ps={1}",
                         request.QualityProfileKey, request.PageSize); // TODO: should handle optional params
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this 'TODO' comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=csharpsquid%3AS1135)